srml/balances/src/lib.rs

```diff
@@ -355,13 +488,14 @@ impl<T: Trait> Module<T> {
 		}
 	}
 
-	/// Increase TotalIssuance by Value.
+	/// Increase stake by `value`.
```

Comment:
This is not an improvement in documentation. In fact, we should rename this function in my opinion

Reply:
> we should rename this function

Agree, but not for this PR

Reply:
ok, but should we put issuance instead of stake in doc in this PR ?

Reply:
Sounds like a chicken and egg problem, I will open an issue which calls this out, and ensures that whatever documentation that does exist will be up to date

Reply:
Stake is removed from docs now (for this module)